### PR TITLE
Add Renfe as hard

### PR DESCRIPTION
**Pull request by Alerymin**
I genuinely need email_body_CODE because of the mess it is. I put it all in english but as there are 3 languages, having one at least in spanish is important

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -14528,6 +14528,22 @@
         ]
     },
 
+    {
+        "name": "Más Renfe",
+        "url": "https://www.renfe.com/es/es/informacion-legal/renfe-operadora/privacidad-cookies/politica-privacidad",
+        "url_es": "https://www.renfe.com/es/en/legal-information/renfe-operadora/privacidad-cookies/politica-privacidad",
```
Comment (tupaschoal) [inline _data/sites.json:14534]: Aren't these two swapped?

```suggestion
        "url": "https://www.renfe.com/es/en/legal-information/renfe-operadora/privacidad-cookies/politica-privacidad",
        "url_es": "https://www.renfe.com/es/es/informacion-legal/renfe-operadora/privacidad-cookies/politica-privacidad",
```

**Review by tupaschoal — COMMENTED** [review]
We don't support localized email bodies, but if the site is Spanish and mostly used by Spanish-speaking folks, I don't mind the body being in Spanish too.

File: _data/sites.json (hunk 1 above)
Comment (Alerymin) [reply _data/sites.json:14534]: Indeed, you are right

**Comment by Alerymin** [comment]
The issue is that even if the website is mainly spanish, it's supposed to be translated to French as they offer service in France to french customers who may not understand spanish, it's already a huge mess that even in French or English the privacy policy is not translated.
I put the email body in Spanish hoping people will change it if needed.

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**